packages/audio-block/src/components/BlockAttachments.tsx

```diff
@@ -0,0 +1,80 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Asset, useBlockAssets, useBlockSettings } from '@frontify/app-bridge';
+import { Attachments, downloadAsset } from '@frontify/guideline-blocks-shared';
+import { BlockAttachmentsProps, BlockSettings } from '../types';
+import { ATTACHMENTS_SETTING_ID } from '../settings';
+
+export const BlockAttachments = ({ assetToDownload, appBridge }: BlockAttachmentsProps) => {
+    const [blockSettings, setBlockSettings] = useBlockSettings<BlockSettings>(appBridge);
```

Comment:
As we talk about assets, settings should appear here.

Reply:
replaced by useAttachments hook, [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/e5a4c8cf4730f2aa65b2522596991235d4d9535e)